ui/src/containers/HeaderContainer.js

```diff
@@ -1,13 +1,18 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { withRouter } from 'react-router-dom';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import config from '../config/config';
 import Header from '../components/Header';
+import { getUser } from '../actions/actions-ui';
 
-const HeaderContainer = ({ currentUser, history }) => {
+const HeaderContainer = ({ currentUser, isUserRequestFailed, history, getUser }) => {
   const [isDropDownOpen, setIsDropDownOpen] = useState(false);
 
+  useEffect(() => {
+    getUser();
+  }, []);
```

Comment:
eslint gives the following warning:

> React Hook useEffect has a missing dependency: 'getUser'. Either include it or remove the dependency array. If 'getUser' changes too often, find the parent component that defines it and wrap that definition in useCallback.eslint(react-hooks/exhaustive-deps)

Reply:
@craicoverflow this one will be resoved as part of this JIRA: https://issues.jboss.org/browse/AEROGEAR-9050 Resolving for now.